gateway/assetlist.json

```diff
@@ -504,6 +504,48 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
       }
     },
+    {
+      "description": "Avalanche USD Coin (Wormhole), Avalanche USDC, factory/wormhole14ejqjyq8um4p3xfqj74yld5waqljf88fz25yxnma0cngspxe3les00fpjx/5ZLmAZpcbaP4EGyihSmpfwryzDr84h51tboV392BCjW4",
+      "denom_units": [
+        {
+          "denom": "factory/wormhole14ejqjyq8um4p3xfqj74yld5waqljf88fz25yxnma0cngspxe3les00fpjx/5ZLmAZpcbaP4EGyihSmpfwryzDr84h51tboV392BCjW4",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "wormhole/5ZLmAZpcbaP4EGyihSmpfwryzDr84h51tboV392BCjW4/6",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "address": "wormhole1utjx3594tlvfw4375esgu72wa4sdgf0q7x4ye27husf5kvuzp5rsr72gdq",
+      "base": "factory/wormhole14ejqjyq8um4p3xfqj74yld5waqljf88fz25yxnma0cngspxe3les00fpjx/5ZLmAZpcbaP4EGyihSmpfwryzDr84h51tboV392BCjW4",
+      "display": "wormhole/5ZLmAZpcbaP4EGyihSmpfwryzDr84h51tboV392BCjW4/6",
+      "name": "Avalanche USD Coin (Wormhole)",
+      "symbol": "avalanche.USDC.wh",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "osmosis",
```

Comment:
In this gateway file, the trace here should point to avalanche USDC (not to Osmosis). Luckily, Avalanche USDC has already been registered to the Chain Registry, so I think you just need to update the chain_name and base_denom here.

Reply:
ok, just changed it